setup.py

```diff
@@ -21,7 +21,7 @@
 # Change docs/sphinx/conf.py too!
 VERSION = '0.4.0'
 
-REQUIRES = ['cliff']
+REQUIRES = ['six', 'cliff']
```

Comment:
Not sure whether it's worth including `six` dependency just to check python versions, but as it's usually installed anyway I'm fine with that...